src/main/scala/com/nthportal/versions/v2/Version.scala

```diff
@@ -21,8 +21,7 @@ final case class Version(major: Int, minor: Int) extends VersionBase[Version, Ex
 
 object Version extends VersionCompanion[Version, ExtendedVersion] with Of[Dot[Version]] {
   override private[versions] val ordering: Ordering[Version] =
-    Ordering.by[Version, Int](_.major)
-      .thenBy(_.minor)
```

Comment:
what's the rationale behind this change?

Incidentally, I need to rename it to `orElseBy` in `extra-predef` to match 2.13, but that's a separate issue

Reply:
nvm, I see I deleted those methods in my overhaul. oops.

this is what happens when you commit code that's more than a year old without looking at it.

I will release `extra-predef` 2.1.0 today or tomorrow to add back `orElse` and `orElseBy` methods (those were the names that were decided on for 2.13).

Reply:
`extra-predef` 2.1.0 is on its way to Maven Central. It adds `orElseBy` to its 2.12 version, so if you wouldn't mind reverting these changes and swapping `thenBy` for `orElseBy`, I would greatly appreciate it. or, if you want, I can take care of that. sorry for the trouble

Reply:
Done 👍  Just out of curiosity, though, what's the benefit of basing Ordering on Ordering composition methods as opposed version tuples in this case? 🤔